packages/renderer/src/lib/volume/VolumeColumnAge.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+import type { VolumeInfoUI } from './VolumeInfoUI';
+
+export let object: VolumeInfoUI;
```

Comment:
Could we pass the value only (age), so we could have a more generic and reusable AgeColumn, or DurationColumn, or NumberColumn?

Reply:
I would like to be able to do this, and share code, but I am worried about the scope and co-dependencies of these PRs and how much we continue to refine before anything is usable and more widely available in main.

To me we always need to support the current behaviour (since columns like name are often custom or include multiple properties) and we could support this later through something like a column type mapping function, e.g. from:
```
  renderer: ContainerColumnAge
```
to:
```
  property: container => container.age,
  renderer: AgeColumn
```